src/app/[locale]/(pages)/session/page.tsx

```diff
@@ -1,85 +1,91 @@
 'use client';
 
-import { Grid, Button, Box, Typography, Card } from '@mui/material';
-import Link from 'next/link';
+import { Box, Button, Card, Grid, Typography } from '@mui/material';
 import { IllusSms } from '@pagopa/mui-italia';
-import { Introduction } from '../../_component/introduction/introduction';
-import { commonBackgroundLight } from '../../_utils/styles';
+import { useTranslations } from 'next-intl';
+import Link from 'next/link';
 import { FAQ } from '../../_component/accordion/faqDefault';
+import { Introduction } from '../../_component/introduction/introduction';
 import { ROUTES } from '../../_utils/routes';
+import { commonBackgroundLight } from '../../_utils/styles';
 
-const Session = (): React.ReactElement => (
-  <>
-    <Grid sx={commonBackgroundLight} container>
-      <Grid item xs={12} justifySelf={'center'}>
-        <Introduction
-          title={'Ciao Mario!'}
-          summary={
-            <>
-              <span>
-                Al momento hai attiva una sessione su app IO con il dispositivo{' '}
-                <strong>iPhone 12 Pro</strong>, se lo hai perso o non lo riconosci esci dal tuo
-                account. Potrai accedere nuovamente da qualsiasi dispositivo.
-              </span>
-            </>
-          }
-          summaryColumns={{ xs: 12, md: 6 }}
-        />
-      </Grid>
+const Session = (): React.ReactElement => {
+  const t = useTranslations('ioesco');
+  return (
+    <>
+      <Grid sx={commonBackgroundLight} container>
+        <Grid item xs={12} justifySelf={'center'}>
+          <Introduction
+            title={t('common.hello', { nome: 'Mario' })}
+            // TODO: [TL] tags
+            summary={
+              <>
+                <span>
+                  {t('lplogoutpostlogin.activesession', { deviceModel: 'iPhone 12 Pro' })}
+                  {/* Al momento hai attiva una sessione su app IO con il dispositivo{' '}
+                  <strong>iPhone 12 Pro</strong>, se lo hai perso o non lo riconosci esci dal tuo
```

Comment:
why there is this comment?

Reply:
I left the previous paragraphs commented as a memo to compare where and which are the tags we have to put in the text. Every one of these has a TODO to fix it when we'll have the correct texts with tags. 

Can you give me a new task so I can change the TODOs into FIXMEs with the new task number in order to pass this PR and come back later on these?